packages/cosmic-swingset/tools/test-kit.js

```diff
@@ -144,54 +153,62 @@ const baseConfig = harden({
  *   any changes
  * @param {Replacer<SwingSetConfig>} [options.fixupConfig] a final opportunity
  *   to make any changes
+ * @param {import('@agoric/swingset-vat/tools/run-utils.js').RunHarness} [options.harness] - Optional run harness
+ * @param {((destPort: string, msg: unknown) => unknown)} [options.mockBridgeReceiver]
  * @param {import('@agoric/telemetry').SlogSender} [options.slogSender]
  * @param {import('../src/chain-main.js').CosmosSwingsetConfig} [options.swingsetConfig]
  * @param {import('@agoric/swing-store').SwingStore} [options.swingStore]
  *   defaults to a new in-memory store
  * @param {SwingSetConfig['vats']} [options.vats] extra static vat configuration
- * @param {string} [options.baseBootstrapManifest] identifies the colletion of
- *   "behaviors" to run at bootstrap for creating and configuring the initial
- *   population of vats (see
- *   {@link ../../vats/tools/bootstrap-chain-reflective.js})
- * @param {string[]} [options.addBootstrapBehaviors] additional specific
- *   behavior functions to augment the selected manifest (see
- *   {@link ../../vats/src/core})
- * @param {string[]} [options.bootstrapCoreEvals] code defining functions to be
- *   called with a set of powers, each in their own isolated compartment
  * @param {object} [powers]
  * @param {Pick<import('node:fs/promises'), 'mkdir'>} [powers.fsp]
  * @param {typeof import('node:path').resolve} [powers.resolvePath]
  */
 export const makeCosmicSwingsetTestKit = async (
-  receiveBridgeSend,
   {
     // Options for the SwingSet controller/kernel.
     bundleDir = 'bundles',
     bundles,
-    configOverrides,
-    defaultManagerType,
+    configSpecifier,
     debugName,
+    defaultManagerType,
     env = process.env,
     fixupInitMessage,
     fixupConfig,
+    harness,
+    mockBridgeReceiver = makeMockBridgeKit(),
```

Comment:
I think the name of this option can be improved to clarify what it does.
```suggestion
    handleBridgeSend = makeMockBridgeKit(),
```